example/simple_webcam.py

```diff
@@ -0,0 +1,34 @@
+from pyacq import create_manager, ImageViewer
+from pyqtgraph.Qt import QtCore, QtGui
+
+
+# create a device in a new pocess
+man = create_manager()
+
+nodegroup = man.create_nodegroup()
+#dev = nodegroup.create_node('WebCamImageIO', name = 'cam0')
+dev = nodegroup.create_node('WebCamAV', name = 'cam0')
+
+dev.configure(camera_num = 0)
+stream_dict = dict(protocol = 'tcp', interface = '127.0.0.1', port = '9000',
+                    transfertmode = 'plaindata', streamtype = 'video',
```

Comment:
Should not need to specify stream type here--the node already knows (and enforces) this.